cluster/src/main/java/com/hartwig/pipeline/sbpapi/SbpRestApi.java

```diff
@@ -108,9 +98,9 @@ public void postFile(final SbpFileMetadata metaData) {
         }
     }
 
-    private void patch(final String sampleID, final String status, final String json, final String entityType) {
-        LOGGER.info("Patching {} id [{}] with status [{}]", entityType, sampleID, status);
-        Response response = api().path(entityType)
+    private void patch(final String sampleID, final String status, final String json) {
+        LOGGER.info("Patching {} id [{}] with status [{}]", SbpRestApi.RUNS, sampleID, status);
+        Response response = api().path(SbpRestApi.RUNS)
```

Comment:
can get rid of the SbpRestApi in SbpRestApi.RUNS? or do you find it more readable?